control/trajectory_follower/design/mpc_lateral_controller-design.md

```diff
@@ -132,8 +167,16 @@ AutonomouStuff Lexus RX 450h for under 40 km/h driving.
 - `weight_terminal_lat_error`: Preferable to set a higher value than normal lateral weight `weight_lat_error` for stability.
 - `weight_terminal_heading_error`: Preferable to set a higher value than normal heading weight `weight_heading_error` for stability.
 
+## References / External links
+
+<!-- Optional -->
+
+- [1] Jarrod M. Snider, "Automatic Steering Methods for Autonomous Automobile Path Tracking",
+  Robotics Institute, Carnegie Mellon University, February 2009.
+
 ## Related issues
 
 <!-- Required -->
 
 - <https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/-/issues/1057>
+- <https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/-/issues/1058>
```

Comment:
You can remove AutowareAuto issues.

Reply:
removed in https://github.com/autowarefoundation/autoware.universe/pull/1591/commits/d25e0d9dc4cf8617098ea922f1514177e4e87113